pkg/syncer/mutators/interface_test.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
wrong filename?

Reply:
please review the new deployment_test file.